src/components/Home/Home.js

```diff
@@ -0,0 +1,35 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import Carousel from 'react-multi-carousel';
+import { fetchMotorcycles } from '../../redux/motorcycles/motorcycles';
+import Motor from './Motor';
+import responsive from '../../config/responsive';
+
+const Home = () => {
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(fetchMotorcycles());
+  }, []);
+  const motors = useSelector((state) => state.motorcycle);
+  return (
+    <div>
+      <h2 className="text-center m-4">POWER MOTORS LATEST MODELS</h2>
+      <p className="text-center header-text2 m-2">Please select a power models</p>
+      <div className="show-motor w-100">
+        <Carousel
```

Comment:
The Carousel is preventing the page from displaying from my end. I think you should check that.